### ISSUE TITLE
recent github version breaks import::* within functions

### ISSUE THREAD

**Issue by mmuurr** [rticulate/import#21]
Unfortunately, I don't know exactly which change corresponds to this behavior, but on the current github version (nominally 1.1.1), this fails:
```R
 > tempf <- function() { import::here("%>%", .from = "magrittr"); 1:10 %>% rev }
 > tempf()
 Error in 1:10 %>% rev (from #1) : could not find function "%>%"
 > packageVersion("import")
 [1] ‘1.1.1’
```
When I revert to 1.1.0 via CRAN, that basic test succeeds:
```R
 > tempf <- function() { import::here("%>%", .from = "magrittr"); 1:10 %>% rev }
 > tempf()
  [1] 10  9  8  7  6  5  4  3  2  1
 > packageVersion("import")
 [1] ‘1.1.0’
```
No other package or R version (3.5.1) changes between these two tests.

I suppose I could walk back along each commit to try to find the point at which this behavior changed ... just let me know if you'd like that help and I can do some commit-based testing.

Cheers!

**Comment by mmuurr** [comment]
The commit that breaks `import` for me is 01c27bf. Is this version (1.1.1) still under testing/development?

**Comment by smbache** [comment]
Thanks for spotting this - that's a bug. Please have a look here https://github.com/smbache/import/pull/22 to see if that fixes it for you.

**Comment by mmuurr** [comment]
Hi! This popped up again on my radar re: #28. I swapped to that alternate branch and indeed, it fixes the problem. I'll continue to use that branch, but perhaps it should be merged with master (along with a patch version bump) for R package managers?

Cheers and thanks!

**Comment by torfason** [comment]
This should now be fixed on master. If you could install the most current version and test whether the fix works for you (and if the new version works for you in general) that would be greatly appreciated. You can get the fix with:
`remotes::install_github("smbache/import")`

[ fixed-in-master : Issue to be closed following CRAN release ]

**Comment by torfason** [comment]
Fixed in release 1.2.0
